src/main/java/org/mitre/synthea/world/agents/Provider.java

```diff
@@ -439,7 +439,8 @@ private Clinician generateClinician(long clinicianSeed, Random clinicianRand,
       long clinicianIdentifier, Provider provider) {
     Clinician clinician = null;
     try {
-      Demographics city = location.randomCity(clinicianRand);
+      Person doc = new Person(clinicianIdentifier);
```

Comment:
Should this be Person(clinicianSeed) instead?

Reply:
No. I admit that it is a little hard to follow, but `clinicianSeed` is the seed for ALL of the clinicians that can be passed in with the GeneratorOptions. The clinicianIdentifier is the seed for THIS particular clinician.

I'll make some changes in this area when I refactor with a `RandomProvider` as suggested above.